### DISCUSSION TITLE
Turn off Mute switch without Google Nest Mini

### DISCUSSION THREAD

**Discussion by jook-network** [tetele/onju-voice-satellite#96] [Q&A] [answered]
I have this Onju Voice board since a couple of days. With the help of some YouTube vids I was able to install the firmware and OTA updates is also working. A Google Nest Mini is on it's way, so I was hoping to try the board without the Nest Mini. But the Mute switch is set to on apparently. And I can't seem to find I way to turn it off. In the logs when I turn off and on the "Use Wake Word" settings. I get this in the output in ESPHome:

[10:47:30][D][switch:012]: 'Use Wake Word' Turning ON.
[10:47:30][D][switch:055]: 'Use Wake Word': Sending state ON
[10:47:30][I][turn_on_wake_word:522]: Trying to start listening for wake word, but mute switch is on

**Comment by tetele** [comment, marked as answer]
> And I can't seem to find I way to turn it off.

Well... there isn't one without the physical switch :) However, you can adapt the config to make the switch pin inverted, which will make it think it's off by default.

[Here's](https://github.com/tetele/onju-voice-satellite/blob/88e2d6117b944fa2bf4fe8c70d167e76743071f2/esphome/onju-voice-microwakeword.yaml#L423) where you need to edit. You can simply add [`inverted: true`](https://www.esphome.io/guides/configuration-types#config-pin-schema) to that pin definition, but ⚠️ DON'T FORGET TO REMOVE IT ONCE YOU PUT THE PCB IN THE ACTUAL UNIT ⚠️

**Reply by jook-network** [reply]
Great that worked!!!
